docs/reference/inference/service-elasticsearch.asciidoc

```diff
@@ -59,7 +76,9 @@ It can be the ID of either a built-in model (for example, `.multilingual-e5-smal
 
 `num_allocations`:::
 (Required, integer)
-The total number of allocations this model is assigned across machine learning nodes. Increasing this value generally increases the throughput.
+The total number of allocations this model is assigned across machine learning nodes.
+Increasing this value generally increases the throughput.
+If `adaptive_allocation` is enabled, do not set this value.
```

Comment:
maybe give a short explanation, like "(...), because it's automatically set." or so